packages/haiku-serialization/src/bll/Bytecode.js

```
@@ -524,6 +524,8 @@ Bytecode.reinitialize = (folder, relpath, bytecode = {}, config = {}) => {
 
   Bytecode.upsertDefaultProperties(bytecode, contextHaikuId, {
     'style.WebkitTapHighlightColor': 'rgba(0,0,0,0)',
+    'style.transformStyle': 'preserve-3d',
+    'style.perspective': '500px',
```

I sort of thought we would put this in `core` rendering instead of making it addressable in the timeline. Thoughts?

You could still address it…if you knew how. But I wouldn't want users to start mucking around with it and then filing `translation.z` types of bugs.

---

It's an explicit feature request to make this addressable so users can control perspective.

---

Couldn't we at least not address `style.transformStyle`? Or do we explicitly want users to be able to type "flat" in too?

---

IMHO, we do.(In any case, code that simply sets the value in the bytecode by default; if we really do want to hide it in the UI, then that would be in a different place in the code. I mention this because even if we do hide it in the UI, I'd still strongly favor making it an explicit property, rather than an implicit/default, or hardcoded somewhere in core.)

---

SG. Great power, great responsibility, etc.